NoSoft-SA/nspack
# hotfix: fix allocate_pallet_check

Summary of this change:

fixed allocation check where a shipped pallet could be allocated to a new load


CHANGELOG entry:
### Fixed
- 

How to test this code via the UI:
use the text box on allocate load to try and add a shipped pallet. it should raise a warning.
